lib/minitest_to_rspec/input/model/klass.rb

```diff
@@ -78,23 +82,24 @@ def test_case?
           active_support_test_case? ||
             action_controller_test_case? ||
             action_mailer_test_case? ||
+            test_unit_test_case? ||
             draper_test_case?
         end
 
         private
 
-        def ancestor_name(exp, index)
-          assert_sexp_type(:colon2, exp)
-          ancestor = exp[index + 1]
-          sexp_type?(:const, ancestor) ? ancestor[1] : ancestor
+        def ancestor_names(exp)
+          return [exp] if exp.is_a?(Symbol)
+
+          sexp_type?(:colon2, exp) || sexp_type?(:const, exp) ||
+            raise(TypeError, "Expected :const or :colon2, got #{exp.inspect}")
+
+          exp.sexp_body.flat_map { |entry| ancestor_names(entry) }
```

Comment:
Normally I'm not a fan of recursion, but I think it looks great here. I tried writing an iterative version and it was quite ugly.